services/net/filemonitor/appsettings.json

```diff
@@ -21,6 +21,9 @@
       "Authority": "https://oidc.gov.bc.ca/auth/realms/gcpe",
       "Audience": "tno-service-account",
       "Secret": "{DO NOT STORE SECRET HERE}"
+    },
+    "Sftp": {
```

Comment:
Implemented this way would be one single password for the service.  The password is more tightly related to the connection.

Reply:
I wasn't using this anymore. I just forgot to remove the Sftp entry.